### ISSUE TITLE
Bug report must correctly handle `System.AggregateException`

### ISSUE THREAD

**Issue by RussKie** [gitextensions/gitextensions#7902]
## Current behaviour

![image](https://user-images.githubusercontent.com/4403806/77609841-2fadd680-6f75-11ea-9420-fb592ae11a3a.png)

## Expected behaviour

The bug report must correctly process `System.AggregateException` and report all inner exceptions

## Steps to reproduce

Modify the code to throw an `System.AggregateException`.
In my case, my repo got corrupted with HEAD and index files containing `NUL`s. Having repaired the super-repo, I was going through the submodules when this happened.

## Environment

- GitExtensions version:
- GIT version: 
- OS version: 
- .NET version: 

## Diagnostics

- Git Extensions 3.3.1.7897
- Build 5a97671645532bcedc443bac7b727f40db47cb5c
- Git 2.21.0.windows.1 (recommended: 2.23.0 or later)
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4150.0
- DPI 96dpi (no scaling)

May be related to #7570

**Comment by RussKie** [comment]
A way to simulate: replace a good config (make a backup!) of a submodule with [this one](https://github.com/gitextensions/gitextensions/files/4385030/config.txt) and open the submodule

**Comment by gerhardol** [comment]
Duplicate of #6762 ?
I cannot reproduce with the patch file. 
The error message is incorrect (presented as a file), the parsing of git-diff fails. 
Not sure how to present this actually, may consider this after #7899.

![image](https://user-images.githubusercontent.com/6248932/77669955-d39f7e00-6f85-11ea-8829-9d20b2df41d3.png)

```
$ git -c diff.submodule=short -c diff.noprefix=false diff --no-color -M -C -z --name-status
fatal: bad config line 1 in file F:/dev/gc/gitextensions/.git/modules/GitExtensionsDoc/config
fatal: 'git status --porcelain=2' failed in submodule GitExtensionsDoc
```

**Comment by RussKie** [comment]
Try open the submodule.

In my case all git configs and HEADs were nulled out.... nfi why

**Comment by gerhardol** [comment]
> Try open the submodule.

That is reported as a separate exception: Could not load config file: F:\dev\gc\gitextensions\.git\modules\GitExtensionsDoc\config
Handled as an unresolved conflict for some reason

I fixed the parsing of git-diff, git-status to not interpret the error messages as file names, but this will hide the problem to the user - there is no sane way to report the problem back to the user form so the effect is just that the user do not see any indication at all.

**Comment by RussKie** [comment]
I will work on this once I compete the installer.
I still have the broken copy.

**Comment by RussKie** [comment]
Trying it now and can't seem to repro in the latest master:
![image](https://user-images.githubusercontent.com/4403806/80300251-5cf0dd00-87de-11ea-83f1-38fe76e4fcb1.png)

**Comment by RussKie** [comment]
![image](https://user-images.githubusercontent.com/4403806/80300387-92e29100-87df-11ea-9d92-b11c93ddce65.png)
![image](https://user-images.githubusercontent.com/4403806/80300390-9d048f80-87df-11ea-9ba8-2b4332f86f3a.png)

Presume resolved, closing.
